README.md

```diff
@@ -79,7 +79,7 @@ This project was forked from [rbenv-win][3] and modified for [pyenv][1]. It is n
    version      Show the current Python version and its origin
    version-name Show the current Python version
    versions     List all Python versions available to pyenv
-   exec         Runs an executable by first preparing PATH so that the selected Python
+   exec         Runs an executable by first preparing PATH so that the selected Python version's `bin' directory is at the front
```

Comment:
```suggestion
   exec         Runs an executable by first preparing PATH so that the selected 
                Python version's `bin' directory is at the front
```

Reply:
I applied that one for you, I like this `Maintainers are allowed to edit this pull request.` feature.